src/provider/provider_tracking.c

```diff
@@ -124,24 +300,45 @@ umf_result_t umfMemoryTrackerGetAllocInfo(const void *ptr,
         return UMF_RESULT_ERROR_NOT_SUPPORTED;
     }
 
-    if (TRACKER->alloc_segments_map == NULL) {
-        LOG_ERR("tracker's alloc_segments_map does not exist");
-        return UMF_RESULT_ERROR_NOT_SUPPORTED;
+    for (int i = 0; i < MAX_LEVELS_OF_ALLOC_SEGMENT_MAP; i++) {
```

Comment:
Do we have to do that on every get?

Reply:
I think that checking `TRACKER->alloc_segments_map[0]` is enough (on every get - as it was so far)